msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -87,15 +87,54 @@ Get started with Teams AI library using the **ChefBot** sample. It enables your
 
 1. After the dependencies are installed, select **File** > **Open Folder**.
 
-1. Go to **teams-ai > js > samples> 04.ai.a.teamsChefBot** and select **Select Folder**. All the files for the chef bot sample are listed under the **EXPLORER** section in Visual Studio Code.
+1. Go to **teams-ai > js > samples> 03.ai-concepts> c.actionMapping-lightBot** and select **Select Folder**. All the files for the Light bot sample are listed under the **EXPLORER** section in Visual Studio Code.
 
-1. Under **EXPLORER**, duplicate the `sample.env` file  and update the duplicate file to `.env`.
+      # [OpenAI key](#tab/OpenAI-key)
 
-1. In the sample folder, update the following code in the `.env` configuration file:
+      Go to `env` folder and update the following code in `./env/.env.local.user` file:
 
-   ```text
-   OPENAI_KEY=<your OpenAI key>
+      ```text
+      SECRET_OPENAI_KEY=<your OpenAI key>
+
+      ```
+
+      # [Azure OpenAI](#tab/Azure-OpenAI)
+
+      1. Go to `env` folder and update the following code in `./env/.env.local.user` file:
+
+         ```text
+         SECRET_AZURE_OPENAI_KEY=<your Azure OpenAI key>
+         SECRET_AZURE_OPENAI_ENDPOINT=<your Azure OpenAI Endpoint>
+
+         ```
+
+      1. Go to `teamsapp.local.yml` file and modify the last step to use Azure OpenAI variables:
+
+         ```text
+         - uses: file/createOrUpdateEnvironmentFile
+         with:
+            target: ./.env
+            envs:
+            BOT_ID: ${{BOT_ID}}
+            BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
+            #OPENAI_KEY: ${{SECRET_OPENAI_KEY}}
+            AZURE_OPENAI_KEY: ${{SECRET_AZURE_OPENAI_KEY}}
+            AZURE_OPENAI_ENDPOINT: ${{SECRET_AZURE_OPENAI_ENDPOINT}}
+         ```
 
+      ---
+
+1. Go to `infra` folder and comment out lines from 72 to 75 and uncomment lines from 76 to 83 in `azure.bicep` file.
```

Comment:
I think it makes more sense to show the code that needs to be commented out instead of giving the line numbers. It might be misleading.